### PR TITLE
[Frontend] bring your own mnemonic ethereum

### PR DESCRIPTION
**Pull request by Joe-mcgee**
Copy pasting addresses and private keys out of the terminal can be a tad bit arduous, this extends the UI folders wonderful build system to allow the option to bring your own Mnemonic that one can put in their meta mask to have all 10 accounts just show up automatically, for speed.

### CODE REVIEW

**Comment by Joe-mcgee** [comment]
https://github.com/Sifchain/sifnode/blob/3c20c1a0ddac7e16974c127a5358072f3aecb9e2/.codecov.yml#L45

hmm should I add - "ui" to this list to bump that codecov up?

File: ui/chains/ethereum/run-ethereum-test-chain.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+MNEMONIC=$(grep MNEMONIC .env | cut -d '=' -f 2-)
+BLOCKSPEED=0
+ganache-cli -m "$MNEMONIC"  -b "$BLOCKSPEED"
```
Comment (ryardley) [inline ui/chains/ethereum/run-ethereum-test-chain.sh:5]: Might be a good idea to run `yarn ganache-cli`. That was we don't need to necessarily install it globally.

**Review by ryardley — CHANGES_REQUESTED** [review]

**Review by ryardley — DISMISSED** [review]

File: ui/chains/ethereum/run-ethereum-test-chain.sh (hunk 1 above)
Comment (Joe-mcgee) [reply ui/chains/ethereum/run-ethereum-test-chain.sh]: https://github.com/Sifchain/sifnode/blob/7770158cfef548ba553e1dc04d7ba3ec8e557b86/ui/chains/ethereum/run-ethereum-test-chain.sh#L5

**Comment by utx0** [comment]
Hey @ryardley @Joe-mcgee where is this bad boy/girl at?

**Comment by ryardley** [comment]
Sorry we haven't gotten on to this PR but we were working off the UI-CLP branch for ages where we hardcode the seed phrase for testing so this hasn't been a priority as that kind of just works. I merged in the work from develop and amended this so the phase defaults to what we have been testing against.

**Review by ryardley — APPROVED** [review]

**Comment by ryardley** [comment]
@59023g you happy with this? If so we can probably merge it.

**Review by 59023g — APPROVED** [review]

**Comment by utx0** [comment]
Hey guys is this code useful at all? Or is it stale now? If not useful lets close the PR out. @59023g @ryardley

**Comment by ryardley** [comment]
Honestly it is not that useful right now as we need deterministic accounts for our testbed.

**Pull request closed without merging**